pttrack/models.py

```diff
@@ -371,9 +371,10 @@ def __unicode__(self):
 
 class Document(Note):
     title = models.CharField(max_length=200)
-    image = models.ImageField(
+    image = models.FileField(
         help_text="Please deidentify all file names before upload!",
```

Comment:
I think that is an important reminder when handling these documents. That way the file on their harddrive will also be deidentified and they remmber to do it every time.